test/detection/test_polygonzone.py

```diff
@@ -0,0 +1,102 @@
+from contextlib import ExitStack as DoesNotRaise
+
+import numpy as np
+import pytest
+
+import supervision as sv
+
+DETECTION_BOXES = np.array(
+    [
+        [35.0, 35.0, 65.0, 65.0],
+        [60.0, 60.0, 90.0, 90.0],
+        [85.0, 85.0, 115.0, 115.0],
+        [110.0, 110.0, 140.0, 140.0],
+        [135.0, 135.0, 165.0, 165.0],
+        [160.0, 160.0, 190.0, 190.0],
+        [185.0, 185.0, 215.0, 215.0],
+        [210.0, 210.0, 240.0, 240.0],
+        [235.0, 235.0, 265.0, 265.0],
+    ],
+    dtype=np.float32,
+)
+
+DETECTIONS = sv.Detections(
```

Comment:
Consider using the `mock_detections` function here. You can find it in `test/test_utils.py`.